src/Test/Desktop/OpenRiaServices.Common.DomainServices.Test/Cities/CityDomainService.cs

```diff
@@ -135,6 +135,18 @@ public async Task<IEnumerable<Zip>> GetZipsAsEnumerable()
             return await Task.FromResult(GetZips());
         }
 
+        [Query(IsComposable = false)]
+        public async Task<Zip> GetZipByFourDigitCodeAsync(int fourDigitCode)
```

Comment:
I suggest you remove async here so you don't need to await the result

Reply:
That's true! I have removed it now.